iotileemulate/iotile/emulate/virtual/peripheral_tile.py

```diff
@@ -62,7 +62,11 @@ def __init__(self, address, device):
 
         self._logger.info("Tile %s at address %d is starting from reset", self.name, self.address)
 
-        address, run_level, debug = await self._device.emulator.await_rpc(8, rpcs.REGISTER_TILE, *self._registration_tuple())
+        try:
+            address, run_level, debug = await self._device.emulator.await_rpc(8, rpcs.REGISTER_TILE, *self._registration_tuple())
+        except:
```

Comment:
This bare except might also want a pylint:disable

Reply:
I just checked this because I was wondering why pylint wasn't flagging it.  `pylint` doesn't care about `bare-except` if you `raise` the exception again since you're not chomping it so this is fine without a `pylint:disable`

Reply:
Oh, good to know!